### ISSUE TITLE
2FA

### ISSUE THREAD

**Issue by NikilXYZ** [makiprogrammer/ts-messenger-api#7]
Allow for authentication using 2Factor (I'm assuming this will be 2 awaits (one for initial login and one for user to type in the 2FA code and wait till it's accepted?)

**Comment by makiprogrammer** [comment]
You are right, it will be two awaits. However, the whole login process is not as easy as that. I spend a lot of time implementing just regular login. The 2FA will take also a lot of effort.

Although 2FA is a thing we want to add to this API, it won't be submitted within a month (as of the date of this comment).
